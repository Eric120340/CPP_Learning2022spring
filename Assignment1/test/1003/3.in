8 2
70 34 
91 92 
76 50 
29 46 
49 41 
39 57 
99 81 
70 20 
2 6
49 36 40 96 27 88 
23 24 86 15 28 48 
