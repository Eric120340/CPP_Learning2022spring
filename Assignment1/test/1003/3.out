4212 3336 5724 7230 2842 7792 
6575 5484 11552 10116 5033 12424 
4874 3936 7340 8046 3452 9088 
2479 2148 5116 3474 2071 4760 
3344 2748 5486 5319 2471 6280 
3222 2772 6462 4599 2649 6168 
6714 5508 10926 10719 4941 12600 
3890 3000 4520 7020 2450 7120 
